app/src/components/annotation/AnnotationSummaryGroup.tsx

```diff
@@ -167,7 +167,7 @@ export const AnnotationSummaryGroupTokens = ({
               clickable
               showClickableIcon={false}
             >
-              {meanScore ? (
+              {meanScore != null ? (
```

Comment:
let's add the eslint rule - I think it will be worth it

Reply:
Pushing to another issue. We violate this rule in a ton of places and I don't want to blow up this PR https://github.com/Arize-ai/phoenix/issues/7440